src/cascadia/TerminalSettingsModel/TerminalSettings.h

```diff
@@ -28,22 +29,40 @@ namespace SettingsModelLocalTests
 
 namespace winrt::Microsoft::Terminal::Settings::Model::implementation
 {
+    struct TerminalSettingsStruct :
+        public TerminalSettingsStructT<TerminalSettingsStruct>
+    {
+    public:
+        TerminalSettingsStruct(Model::TerminalSettings defaultSettings, Model::TerminalSettings unfocusedSettings) :
+            _defaultSettings(defaultSettings),
+            _unfocusedSettings(unfocusedSettings) {}
+
+        Model::TerminalSettings DefaultSettings() { return _defaultSettings; };
+        Model::TerminalSettings UnfocusedSettings() { return _unfocusedSettings; };
+
+    private:
+        Model::TerminalSettings _defaultSettings;
+        Model::TerminalSettings _unfocusedSettings;
+    };
+
     struct TerminalSettings : TerminalSettingsT<TerminalSettings>, IInheritable<TerminalSettings>
     {
         TerminalSettings() = default;
 
-        static Model::TerminalSettings CreateWithProfileByID(const Model::CascadiaSettings& appSettings,
-                                                             guid profileGuid,
-                                                             const Control::IKeyBindings& keybindings);
+        static Model::TerminalSettingsStruct CreateWithProfileByID(const Model::CascadiaSettings& appSettings,
+                                                                   guid profileGuid,
+                                                                   const Control::IKeyBindings& keybindings);
 
-        static Model::TerminalSettings CreateWithNewTerminalArgs(const Model::CascadiaSettings& appSettings,
-                                                                 const Model::NewTerminalArgs& newTerminalArgs,
-                                                                 const Control::IKeyBindings& keybindings);
+        static Model::TerminalSettingsStruct CreateWithNewTerminalArgs(const Model::CascadiaSettings& appSettings,
+                                                                       const Model::NewTerminalArgs& newTerminalArgs,
+                                                                       const Control::IKeyBindings& keybindings);
 
-        static Model::TerminalSettings CreateWithParent(const Model::TerminalSettings& parent);
+        static Model::TerminalSettingsStruct CreateWithParent(const Model::TerminalSettingsStruct& parent);
         void SetParent(const Model::TerminalSettings& parent);
 
         void ApplyColorScheme(const Model::ColorScheme& scheme);
+        void ApplyAppearanceSettings(const Microsoft::Terminal::Settings::Model::IAppearanceConfig& appearance,

```

Comment:
nit: can this be private `_ApplyAppearanceSettings`?

Reply:
Yep!